node/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ All notable changes to this project will be documented in this file.  The format
 * In addition to `consensus` and `deploy_requests`, the following values can now be controlled via the `[network.estimator_weights]` section in config: `gossip`, `finality_signatures`, `deploy_responses`, `block_requests`, `block_responses`, `trie_requests` and `trie_responses`.
 * Nodes will now also gossip deploys onwards while joining.
   Added run-mode field to the `/status` endpoint and the `info_get_status` JSON-RPC.
+* Added a new RPC and REST endpoint that returns the bytes of the `chainspec.toml` file.
```

Comment:
Hmm, can we actually explicitly mention the endpoints here? I think it's `info_get_chainspec` and `/chainspec`?